RELEASES.md

```diff
@@ -0,0 +1,59 @@
+# Release cycle and process
+Velero team is aiming for 3 releases per year, but will commit to two releases per year with few patch releases in between.
+Planing for new release starts with relabeling all candidates that were not completed in the current release - in example [1.11-candidates](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A1.11-candidate) and labeling all the new for the upcoming release.
+
+In parallel we start a [discussion](https://github.com/vmware-tanzu/velero/discussions?discussions_q=label%3ASurvey) to collect new ideas, proposals, needs and new contributors.
+
+# Versioning and Release
+This document describes the versioning and release process of Velero. This document is a living document, contents will be updated according to each releases.
+
+## Releases
+Velero releases will be versioned using dotted triples, similar to [Semantic Version](http://semver.org/). For this specific document, we will refer to the respective components of this triple as `<major>.<minor>.<patch>`. The version number may have additional information, such as "-rc1,-rc2,-rc3" to mark release candidate builds for earlier access. Such releases will be considered as "pre-releases".
+
+### Major and Minor Releases
+Major and minor releases of Velero will be branched from `main` when the release reaches to `RC(release candidate)` state. The branch format should follow `release-<major>.<minor>.0`. For example, once the release `v1.0.0` reaches to RC, a branch will be created with the format `release-1.0.0`. When the release reaches to `GA(General Available)` state, The tag with format `v<major>.<minor>.<patch>` and should be made with command `git tag -s v<major>.<minor>.<patch>`. The release cadence is around 3 months, might be adjusted based on open source event, but will communicate it clearly.
+
+### Patch releases
+Patch releases are based on the major/minor release branch, the release cadence for patch release of recent minor release is one month to solve critical community and security issues. The cadence for patch release of recent minus two minor releases are on-demand driven based on the severity of the issue to be fixed.
+
+### Pre-releases
+`Pre-releases:mainly the different RC builds` will be compiled from their corresponding branches. Please note they are done to assist in the stabilization process, no guarantees are provided.
+
+### Minor Release Support Matrix
+| Version | Supported          |
+| ------- | ------------------ |
+| Velero v1.10.x   | :white_check_mark: |
+| Velero v1.9.x   | :white_check_mark: |
+| Velero v1.8.x   | :white_check_mark: |
```

Comment:
Are we still supporting v1.8.x?

Given the limited resource we have would it be acceptable to support 1.10 and 1.9?

Reply:
Do we? If we don't then will remove it :)

Reply:
that's the reason to have it here so we can have a clear overview :) thanks @reasonerjt

Reply:
In that case we have to define the support cycle ? like current -1  ?

Reply:
> that's the reason to have it here so we can have a clear overview :)
Absolutely thanks for writing this up!

Per my understanding `current` is the ongoing one, as of now it's v1.11, so by `n-2` we support `v1.10.x` and `v1.9.x`

I hope we only support these 2 minor versions whether we call it `n-1` or `n-2`, but anyway I'm open for discussion if there's any disagreement.

Reply:
I don't know about that. 1.11 doesn't exist as a release yet. 1.10 is the *current* release. We don't support 1.11 right now, since we don't support not-yet-released versions. I think in this case "n-2" would be generally understood to mean "the latest released version, and the two before it".

Reply:
@OrlinVasilev I had a discussion with @yanji09 and confirmed we'll need to maintain `n-2`, i.e before v1.11 is out we'll support v1.8.x also.

Resoving this comment

Reply:
@OrlinVasilev I had a discussion with @yanji09 and confirmed that before v1.11 is out we'll also need to support v1.8, therefore I'm resolving this comment.